gruntfile.js

```diff
@@ -10,7 +10,14 @@ module.exports = function(grunt)
                     basePath: 'themes',
                     declaration: false
                 },
-                src: ['src/default/assets/js/src/**/*.ts'],
+                src: [
+                    'src/default/assets/js/src/lib/**/*.ts',
+                    'src/default/assets/js/src/typedoc/Application.ts',
+                    'src/default/assets/js/src/typedoc/components/**/*.ts',
+                    'src/default/assets/js/src/typedoc/services/**/*.ts',
+                    'src/default/assets/js/src/typedoc/utils/**/*.ts',
```

Comment:
oh, i forgot. i had thought about it before. the issue is that `Application.ts` has to be concated before `components` and `services`, because the register functions called by the latter ones are defined there.